rate_counter/rate/utils.py

```diff
@@ -0,0 +1,19 @@
+def get_color(result):
```

Comment:
@Scandie what do you think - does it make sense to write unit tests for your utils in this ticket or create separate one?